src/content/en/2019/css.md

```diff
@@ -0,0 +1,115 @@
+---
+part_number: I
+chapter_number: 2
+title: CSS
+authors: [una, argyleink]
+reviewers: [meyerweb, huijing]
+---
+
+## Introduction
+Cascading Style Sheets are used to paint, format, and layout web pages. Their capabilities span concepts as simple as text color to 3D perspective. It also has hooks to empower developers to handle varying screen sizes, viewing contexts, and printing. CSS helps developers wrangle the content and ensure it's adapting properly the requesting user.
+
+When describing CSS to those not familiar with web technology, it can be helpful to think of it as the language to paint the walls of the house, describe the size and position of windows and doors, as well as flourishing decorations such as wallpaper or plant life. The fun twist to that story, is that depending on who is walking through the house, a developer can adapt to their preferences or contexts!
```

Comment:
"depending on who is walking through the house, a developer can adapt to their preferences or contexts!"
Maybe it's just me, but this reads as if as someone walks through the house, the developer is the one who changes not the house >_<

Reply:
hehe, looks like another spot where 1 word is missing and ruining the meaning. lovely catches 🎣